### PR TITLE
fix(DB/Gameobject): duplicate or wrong spawns

### PR DESCRIPTION
**Pull request by Helias**
## Changes Proposed:
- remove one spawn
- moved the spawn of the herb that is unreachable

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5243
- Closes https://github.com/chromiecraft/chromiecraft/issues/394

## How to Test the Changes:
- .go o 4214

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by pklloveyou** [comment]
Tested

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@Helias 

> 2021-04-11 23:26:55 `pool_gameobject` has a non existing gameobject spawn (GUID: 8871) defined for pool id (959), skipped.

File: data/sql/updates/pending_db_world/rev_1618177302172059302.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1618177302172059302');
+
+DELETE FROM `gameobject` WHERE `id`=2046 AND `guid`=8871;
+
```
Comment (pklloveyou) [inline data/sql/updates/pending_db_world/rev_1618177302172059302.sql:4]:
```sql 
DELETE FROM `pool_gameobject` WHERE `guid` IN (8871);
```

Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1618177302172059302.sql:6]: thanks, added

**Pull request merged**